src/index.html

```diff
@@ -6,9 +6,82 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/helpers/reset.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img
+          src="images/logo.png"
+          alt="logo"
+          width="40px"
+          height="40px"
+          >
```

Comment:
```suggestion
          <img
            src="images/logo.png"
            alt="logo"
            class="logo__image"
          >
```
- Don't use inline styles, use classes for styling instead.
- Pay attention to the checklist: https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md
[CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.